rke/k8s_rke_system_images.go

```diff
@@ -4696,6 +4696,86 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.7",
 		},
+		// Enabled in v2.5.8
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		// Reminder: This template should remain an exact copy of the template for v2.4.x (No ACI images)
```

Comment:
Updated this comment to something more appropriate since the previous one wasn't accurate anymore